public/src/hooks/useModule.ts

```diff
@@ -30,7 +30,7 @@ export const useModule = <T>(path: string, name: string): React.FC<T> | undefine
     window.remoteImport(`${baseUrl}/${path}`).then(bannerModule => {
       setModule(() => withPreviewStyles(bannerModule[name]));
     });
-  }, []);
+  }, [name]);
```

Comment:
[eslint-plugin-react-hooks](https://www.npmjs.com/package/eslint-plugin-react-hooks) has a rule `react-hooks/exhaustive-deps` which lets you know about things your effect depends on but that aren't in the dependency list passed to `useEffect`. Maybe worth adding?